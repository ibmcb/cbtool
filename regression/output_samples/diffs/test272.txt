Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/mrhines/cbtool/lib/operations//../../cbact --procid=TEST_mrhines --osp=port:6379,experiment_inst:TEST_mrhines,host:172.16.1.222,pid:TEST_mrhines,dbid:0,timout:70,protocol:TCP,kind:redis,username:mrhines --msp=cloudoption_testcloud_host:172.16.0.254,timeout:240,cloudoption_testcloud_usage:shared,host:172.16.1.222,port:27017,pid:TEST_mrhines,database:metrics,protocol:TCP,kind:mongodb,username:mrhines --oop=TESTCLOUD,oldest,none --operation=vm-capture --cn=TESTCLOUD --uuid=09F28174-B596-50D0-BF4F-BA9A1596599E --daemon". The process id is 14253.
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |86.64.13.123    |1377115429     |none                                    |089BF014-63ED-521B-9C56-6A2E0CB7E479
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |197.241.39.37   |1377115468     |none                                    |07AC899E-55E1-5C90-BE3A-8BEBC45DC18B
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |54.30.255.166   |1377115485     |none                                    |1C0B4DFB-2660-597B-9885-42A17F481679
|vm_33         |client_daytrader    |nano32      |86.64.13.123    |simhosta4                  |SUT                 |ai_3    |none       |5D0E801B-7B17-5071-9F2F-544DA08807AD
|vm_34         |was                 |iron32      |148.63.9.138    |simhostc2                  |SUT                 |ai_3    |none       |8DFD74DC-119F-5C55-838E-530DE720444D
|vm_35         |db2                 |platinum64  |168.87.73.205   |simhostc4                  |SUT                 |ai_3    |none       |766FB294-B300-5CEA-ADF4-3EA158B9F4F3
|vm_36         |client_daytrader    |nano32      |197.241.39.37   |simhostc1                  |SUT                 |ai_4    |none       |BC108A18-3B66-576E-8895-707EF69F7861
|vm_38         |db2                 |platinum64  |200.122.223.254 |simhostc1                  |SUT                 |ai_4    |none       |D34D700A-54A2-5415-8F14-E87E0E5DF352
|vm_37         |was                 |iron32      |14.61.28.112    |simhostc2                  |SUT                 |ai_4    |none       |68B954F9-0EA7-5BDE-9FBA-96DC72A282AC
|vm_40         |netserver           |iron32      |236.2.66.133    |simhostb3                  |SUT                 |ai_5    |none       |65EDE7C1-CD8A-5CBD-B129-7861E35B8DB6
|vm_39         |netclient           |platinum64  |54.30.255.166   |simhostc3                  |SUT                 |ai_5    |none       |279A40B7-0AA1-57B3-82B5-11ABBCC19914
|Used Memory                                          |20.79M                        
|Uptime (in seconds)                                  |2516690                       
|Total Connections Received                           |185656                        
|Total Commands Processed                             |11640268                      
|Storage Size                                         |18350080                      
|Data Size                                            |6684348                       
|Index Size                                           |2101232                       
|Average Object Size                                  |614.200863732                 
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|1                             
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|0                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|6                             
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|1                             
