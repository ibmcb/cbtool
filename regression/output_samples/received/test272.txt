Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/mrhines/cbtool/lib/operations//../../cbact --procid=TEST_mrhines --osp=port:6379,experiment_inst:TEST_mrhines,host:172.16.1.222,pid:TEST_mrhines,dbid:0,timout:70,protocol:TCP,kind:redis,username:mrhines --msp=cloudoption_testcloud_host:172.16.0.254,timeout:240,cloudoption_testcloud_usage:shared,host:172.16.1.222,port:27017,pid:TEST_mrhines,database:metrics,protocol:TCP,kind:mongodb,username:mrhines --oop=TESTCLOUD,oldest,none --operation=vm-capture --cn=TESTCLOUD --uuid=09F28174-B596-50D0-BF4F-BA9A1596599E --daemon". The process id is 14253.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |86.64.13.123    |1377115429     |none                                    |089BF014-63ED-521B-9C56-6A2E0CB7E479
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |197.241.39.37   |1377115468     |none                                    |07AC899E-55E1-5C90-BE3A-8BEBC45DC18B
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |54.30.255.166   |1377115485     |none                                    |1C0B4DFB-2660-597B-9885-42A17F481679

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |nano32      |86.64.13.123    |simhosta4                  |SUT                 |ai_3    |none       |5D0E801B-7B17-5071-9F2F-544DA08807AD
|vm_34         |was                 |iron32      |148.63.9.138    |simhostc2                  |SUT                 |ai_3    |none       |8DFD74DC-119F-5C55-838E-530DE720444D
|vm_35         |db2                 |platinum64  |168.87.73.205   |simhostc4                  |SUT                 |ai_3    |none       |766FB294-B300-5CEA-ADF4-3EA158B9F4F3
|vm_36         |client_daytrader    |nano32      |197.241.39.37   |simhostc1                  |SUT                 |ai_4    |none       |BC108A18-3B66-576E-8895-707EF69F7861
|vm_38         |db2                 |platinum64  |200.122.223.254 |simhostc1                  |SUT                 |ai_4    |none       |D34D700A-54A2-5415-8F14-E87E0E5DF352
|vm_37         |was                 |iron32      |14.61.28.112    |simhostc2                  |SUT                 |ai_4    |none       |68B954F9-0EA7-5BDE-9FBA-96DC72A282AC
|vm_40         |netserver           |iron32      |236.2.66.133    |simhostb3                  |SUT                 |ai_5    |none       |65EDE7C1-CD8A-5CBD-B129-7861E35B8DB6
|vm_39         |netclient           |platinum64  |54.30.255.166   |simhostc3                  |SUT                 |ai_5    |none       |279A40B7-0AA1-57B3-82B5-11ABBCC19914

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |20.79M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2516690                       
|Total Connections Received                           |185656                        
|Total Commands Processed                             |11640268                      
|Number of Keys                                       |487                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18350080                      
|Data Size                                            |6684348                       
|Index Size                                           |2101232                       
|Average Object Size                                  |614.200863732                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |8                             
|AIs                                                  |3                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |8                             
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |3                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |180                           

 ---------------- VMC-WIDE COUNTERS ----------------
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|1                             
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|0                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|6                             
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|1                             

