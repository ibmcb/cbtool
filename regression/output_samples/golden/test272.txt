Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/mrhines/cbtool/lib/operations//../../cbact --procid=TEST_mrhines --osp=port:6379,experiment_inst:TEST_mrhines,host:172.16.1.222,pid:TEST_mrhines,dbid:0,timout:70,protocol:TCP,kind:redis,username:mrhines --msp=cloudoption_testcloud_host:172.16.0.254,timeout:240,cloudoption_testcloud_usage:shared,host:172.16.1.222,port:27017,pid:TEST_mrhines,database:metrics,protocol:TCP,kind:mongodb,username:mrhines --oop=TESTCLOUD,oldest,none --operation=vm-capture --cn=TESTCLOUD --uuid=FCDBE507-003A-559E-AE9B-0CB69F57FE8C --daemon". The process id is 30399.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |181.5.208.158   |1377110715     |none                                    |D8F96B3D-C7F7-5662-9A96-059789043E4B
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |42.84.21.107    |1377110755     |none                                    |12C14E92-3BBE-57C8-9193-E05723EC9885
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |134.210.131.139 |1377110773     |none                                    |92E57E5D-535F-5DBE-A147-76953867F36D

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |nano32      |181.5.208.158   |simhostc3                  |SUT                 |ai_3    |none       |F5F54937-504E-504E-AE03-DD23E95957E6
|vm_35         |was                 |iron32      |4.97.242.234    |simhostc1                  |SUT                 |ai_3    |none       |79846DFF-7439-52CB-821C-F0CC3B2AA98C
|vm_34         |db2                 |platinum64  |229.183.189.50  |simhosta2                  |SUT                 |ai_3    |none       |AC9701CD-ED26-5DA9-A551-E306613E4701
|vm_36         |client_daytrader    |nano32      |42.84.21.107    |simhostb4                  |SUT                 |ai_4    |none       |A6454ACA-05C4-5B67-ADED-91A8D8269D5B
|vm_37         |was                 |iron32      |157.205.67.232  |simhosta0                  |SUT                 |ai_4    |none       |F551F4C8-DC8A-59AE-9805-6BED29D7313F
|vm_38         |db2                 |platinum64  |170.17.24.192   |simhostc3                  |SUT                 |ai_4    |none       |37C51DE9-5A99-55D4-9EFE-8156BE963529
|vm_39         |netclient           |platinum64  |134.210.131.139 |simhosta0                  |SUT                 |ai_5    |none       |BC798DB3-B1EF-545A-B92D-1A9FCA32CD93
|vm_40         |netserver           |iron32      |190.176.240.31  |simhostc0                  |SUT                 |ai_5    |none       |C1F9997C-A190-5824-84C0-7B2CB7858FBB

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |20.87M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2511978                       
|Total Connections Received                           |185378                        
|Total Commands Processed                             |11120412                      
|Number of Keys                                       |487                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18321408                      
|Data Size                                            |5944156                       
|Index Size                                           |1937712                       
|Average Object Size                                  |619.957863997                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |8                             
|AIs                                                  |3                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |8                             
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |3                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |180                           

 ---------------- VMC-WIDE COUNTERS ----------------
|C4E90296-1043-55C8-8270-B5D7C3C38270 (simzone_b) VM RESERVATIONS|1                             
|2AEE56D4-E6A4-5FB2-8D29-4268D4C22AD0 (simzone_a) VM RESERVATIONS|3                             
|5F641DA9-DE4B-538C-B0CD-42A4F7D43E9E (simzone_d) VM RESERVATIONS|0                             
|A796D59F-B618-5C91-8FC6-F6F093DE22C5 (simzone_c) VM RESERVATIONS|4                             

